### ISSUE TITLE
Deserialization of ES range queries does not work

### ISSUE THREAD

**Issue by fmassot** [quickwit-oss/quickwit#3248]
The DSL is:

```json
"range": {
   "age": {
     "gte": 10,
     "lte": 20,
     "boost": 2.0
   }
}
```

and we expect 

```json
"range": {
   "field": "age",
   "gte": 10,
   "lte": 20,
   "boost": 2.0
}
```

**Comment by fulmicoton** [comment]
Fixed in #3148

**Comment by fmassot** [comment]
Closing as #3148 was merged.
